teps/0003-tekton-catalog-organization.md

```diff
@@ -495,30 +496,42 @@ Infrastructure](https://docs.google.com/document/d/1-czjvjfpuIqYKsfkvZ5RxIbtoFNL
    indicate the name to display in the Tekton Hub, otherwise the name
    will be computed based on the task file name.
 
-4. Resources can include annotation "tekton.dev/tags" which is a comma
+4. Resources must include the annotaion "tekton.dev/category" to indicate the category associated with the resource e.g. git-clone task can have a category as tekton.dev/category: "git"
+
+    1. There should be only one category for a resource. Multiple categories are not allowed.
+
+    2. CI/Catlin should make sure that category is added for the task and there is only one category. Maintaining the category is the responsibility of the author.
+
+    3. If category is not added,Catlin should throw an warning to add a category
```

Comment:
I just saw above that category annotation must be there
```
tekton.dev/category: "git"                 👈 MUST: category of a resource
```
Shouldn't catlin be throwing an error instead of warning if annotation not found? :thinking:

Reply:
yes if the annotation is mandatory then catlin must throw an error. but should it be mandatory?